Release v24.1.0 (#49)
C2Devel/terraform-provider-croccloud: CHANGELOG.md

## 24.1.0

NOTES:

* Rockit Cloud rebranding

FEATURES:

* New Data Source: `aws_ec2_host`

ENHANCEMENTS:

> * resources/aws_route53_record: Remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as it is not supported

Reviewer: ```suggestion
* resources/aws_route53_record: remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as they are not supported
```

с маленькой буквы после `:` и ниже

> * resources/aws_route53_record: remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as it is not supported

Reviewer: ```suggestion
* resource/aws_route53_record: remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as they are not supported
```

> * resources/aws_route53_record: Fix description in documentation about records that contain more than 255 characters

Reviewer: Перемести в `DOCUMENTATION FIXES:`

* datasource/aws_route53_zone: Make the `IsTruncated` parameter optional as it is not supported
* resource/aws_instance: Add `tenancy`, `affinity` and `host_id` arguments support
* resource/aws_launch_template: Add `tenancy`, `affinity` and `host_id` arguments of the `placement` block support

> * data-source/aws_instance: Add `tenancy`, `affinity` and `host_id` attributes support

Reviewer: ```suggestion
* datasource/aws_instance: Add `tenancy`, `affinity` and `host_id` attributes support
```

* resource/aws_ami: update volume type default value

DOCUMENTATION FIXES:

* datasource/aws_ami, datasource/aws_ami_ids: `executable_users` description: add `all` to the list of possible values
* datasource/aws_ebs_snapshot, datasource/aws_ebs_snapshot_ids: `owners` description: add `self` to the list of possible values

> * resources/aws_route53_record: fix description in documentation about records that contain more than 255 characters

Reviewer: ```suggestion
* resource/aws_route53_record: fix description in documentation about records that contain more than 255 characters
```

## 24.1.0

NOTES:

* Rockit Cloud rebranding

FEATURES:

* **New Data Source:** `aws_ec2_host`

ENHANCEMENTS:

* datasource/aws_route53_zone: make the `IsTruncated` parameter optional as it is not supported
* datasource/aws_instance: add `tenancy`, `affinity` and `host_id` attributes support

> * resource/aws_route53_record: remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as it is not supported

Reviewer: ```suggestion
* resource/aws_route53_record: remove the `StartRecordName`, `StartRecordType`, `MaxItems` parameters as they are not supported
```
